### DISCUSSION TITLE
get_flops.py does not support Real-BasicVSR

### DISCUSSION THREAD

**Discussion by Ha0Tang** [open-mmlab/mmediting#1287] [General]

**Comment by zengyh1900** [comment]
Please check this issue. @Z-Fran

**Comment by Z-Fran** [comment]
Sorry for the late reply. Can you supply error logs? I succeed to run `python tools/get_flops.py configs/restorers/real_basicvsr/realbasicvsr_c64b20_1x30x8_lr5e-5_150k_reds.py --shape 15 3 250 250` .
